tests/texlive/latex2html.pm

```diff
@@ -0,0 +1,59 @@
+# SUSE's openQA tests
+#
+# Copyright 2024 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Package: latex2html
+# Summary: Test the  latex2html command with a simple example
+# - Install latex2html
+# - Run late2html command on LaTex document to convert it to HTML
+# - Open created HTML and check
+# - Cleanup
+# Maintainer: QE Core <qe-core@suse.de>
+
+use base 'x11test';
+use x11utils 'ensure_unlocked_desktop';
+use strict;
+use warnings;
+use testapi;
+use utils;
+
+my $latex_data = <<EOF;
+\\documentclass{article}
+\\begin{document}
+Hello, this is a LaTeX sample example page!
+EOF
+sub run {
+    my ($self) = @_;
+    select_console('root-console');
```

Comment:
Why no serial console also for cleanup ?

Reply:
I have added serial console and verified